text/0000-ember-data-packages.md

```diff
@@ -0,0 +1,396 @@
+      <td>import { InvalidError } from '@ember-data/adapters/errors';</td>
+    </tr>
+    <tr>
+      <td>DS.TimeoutError</td>
+      <td>import { TimeoutError } from 'ember-data/adapters/errors';</td>
+      <td>import { TimeoutError } from '@ember-data/adapters/errors';</td>
+    </tr>
+    <tr>
+      <td>DS.AbortError</td>
+      <td>import { AbortError } from 'ember-data/adapters/errors';</td>
+      <td>import { AbortError } from '@ember-data/adapters/errors';</td>
+    </tr>
+    <tr>
+      <td>DS.UnauthorizedError</td>
+      <td>import { UnauthorizedError } from 'ember-data/adapters/errors';</td>
+      <td>import { UnauthorizedError } from '@ember-data/adapters/errors';</td>
+    </tr>
+    <tr>
+      <td>DS.ForbiddenError</td>
+      <td>import { ForbiddenError } from 'ember-data/adapters/errors';</td>
+      <td>import { ForbiddenError } from '@ember-data/adapters/errors';</td>
+    </tr>
+    <tr>
+      <td>DS.NotFoundError</td>
+      <td>import { NotFoundError } from 'ember-data/adapters/errors';</td>
+      <td>import { NotFoundError } from '@ember-data/adapters/errors';</td>
+    </tr>
+    <tr>
+      <td>DS.ConflictError</td>
+      <td>import { ConflictError } from 'ember-data/adapters/errors';</td>
+      <td>import { ConflictError } from '@ember-data/adapters/errors';</td>
+    </tr>
+    <tr>
+      <td>DS.ServerError</td>
+      <td>import { ServerError } from 'ember-data/adapters/errors';</td>
+      <td>import { ServerError } from '@ember-data/adapters/errors';</td>
+    </tr>
+    <tr>
+      <td>DS.errorsHashToArray</td>
+      <td>none</td>
+      <td>import { errorsHashToArray } from '@ember-data/adapters/errors';<br>
+         <br>this public method should also be a candidate for deprecation
+      </td>
+    </tr>
+    <tr>
+      <td>DS.errorsArrayToHash</td>
+      <td>none</td>
+      <td>import { errorsArrayToHashr } from '@ember-data/adapters/errors';<br>
+        <br>this public method should also be a candidate for deprecation
+      </td>
+    </tr>
+    <tr>
+      <td colspan="3"><h3>@ember-data/serializers</h3></td>
+    </tr>
+    <tr>
+      <td>DS.Serializer</td>
+      <td>import Serializer from 'ember-data/serializer';</td>
+      <td>import Serializer from '@ember-data/serializers';</td>
+    </tr>
+    <tr>
+      <td>DS.JSONSerializer</td>
+      <td>import JSONSerializer from 'ember-data/serializers/json';</td>
+      <td>import JSONSerializer from '@ember-data/serializers/json';</td>
+    </tr>
+    <tr>
+      <td>DS.RESTSerializer</td>
+      <td>import RESTSerializer from 'ember-data/serializers/rest';</td>
+      <td>import RESTSerializer from '@ember-data/serializers/rest';</td>
+    </tr>
+    <tr>
+      <td>DS.EmbeddedRecordsMixin</td>
+      <td>import EmbeddedRecordsMixin from 'ember-data/serializers/embedded-records-mixin';</td>
+      <td>import EmbeddedRecordsMixin from 'ember-data/serializers/rest/mixins';</td>
```

Comment:
👋 Shouldn't this be either

- ` import EmbeddedRecordsMixin from 'ember-data/serializers/rest/mixins/embedded-records';`

or

- ` import { EmbeddedRecordsMixin } from 'ember-data/serializers/rest/mixins';`

? Where the first would be more consistent with how we would import the `BuildURLMixin`.

```suggestion
      <td>import EmbeddedRecordsMixin from 'ember-data/serializers/rest/mixins/embedded-records';</td>
```

Reply:
Was a mistake, should have been

```js
import EmbeddedRecordsMixin from '@ember-data/serializers/rest/mixins/embedded-record';
```